dbt_project.yml

```diff
@@ -11,11 +11,11 @@ models:
       +materialized: ephemeral
```

Comment:
if you decide to remove intermediate models, you can remove this for intermediate materialization!